### ISSUE TITLE
Analyze mouse 10x v2 dataset: Mouse Endoderm Project

### ISSUE THREAD

**Issue by jychien** [HumanCellAtlas/secondary-analysis#826]
The "Mouse Endoderm Project" dataset is ready for secondary analysis.

**Project title**: The emergent landscape of the mouse gut endoderm at single-cell resolution
**Project short name**: Mouse Endoderm Project
**Project UUID**: 4e6f083b-5b9a-4393-9890-2a83da8188f1
**Metadata**: mouse, 10x v2
**Bundles**: 29 primary bundles

https://tracker.data.humancellatlas.org/?projectUUID=4e6f083b-5b9a-4393-9890-2a83da8188f1

@HumanCellAtlas/data-ops

**Comment by samanehsan** [comment]
Is analysis for this mouse 10xv2 dataset still blocked? @rexwangcc @jychien

**Comment by jychien** [comment]
@samanehsan @rexwangcc Yes, this dataset is still currently blocked. There was a specimen that did not have a link to the appropriate cell suspension. This fix would require a change to links.json, which is currently not implemented by AUDR simple updates.

https://humancellatlas.slack.com/archives/GACM2JCMS/p1572374545016000
https://humancellatlas.slack.com/archives/C4LTU5RT8/p1572447401090200

**Comment by samanehsan** [comment]
Thanks @jychien! Is the plan to re-ingest this project with the correct linking or to wait until the links.json can be updated? 

Also the first slack link shows me an error page 😞 Was the comment removed?

**Comment by jychien** [comment]
@samanehsan Oh! Looks I linked to a private channel. Here's the beginning of the thread for the second slack discussion that talked about the same thing in ingest:
https://humancellatlas.slack.com/archives/C4LTU5RT8/p1572387812085100

Tentative plans are to re-ingest, but wranglers are still solidifying the current plans. Let me know if you have any other questions. :)

**Comment by samanehsan** [comment]
Ok, thanks for the update! 🙂
